tmt/steps/provision/artemis.py

```diff
@@ -488,65 +489,65 @@ class ProvisionArtemis(tmt.steps.provision.ProvisionPlugin):
         """ Prepare command line options for Artemis """
         return [
-            click.option(
+            option(
                 '--api-url', metavar='URL',
                 help="Artemis API URL.",
                 envvar='ARTEMIS_API_URL'
                 ),
-            click.option(
+            option(
                 '--api-version', metavar='x.y.z',
                 help="Artemis API version to use.",
                 type=click.Choice(SUPPORTED_API_VERSIONS),
                 envvar='ARTEMIS_API_VERSION'
                 ),
-            click.option(
+            option(
                 '--arch', metavar='ARCH',
                 help='Architecture to provision.'
                 ),
-            click.option(
+            option(
                 '--image', metavar='COMPOSE',
                 help='Image (or "compose" in Artemis terminology) '
                      'to provision.'
                 ),
-            click.option(
+            option(
                 '--pool', metavar='NAME',
                 help='Pool to enforce.'
                 ),
-            click.option(
+            option(
                 '--priority-group', metavar='NAME',
                 help='Provisioning priority group.'
                 ),
-            click.option(
+            option(
                 '--keyname', metavar='NAME',
                 help='SSH key name.'
                 ),
-            click.option(
+            option(
                 '--user-data', metavar='KEY=VALUE',
                 help='Optional data to attach to guest.',
                 multiple=True,
                 default=[]
                 ),
-            click.option(
+            option(
                 '--provision-timeout', metavar='SECONDS',
                 help=f'How long to wait for provisioning to complete, '
                      f'{DEFAULT_PROVISION_TIMEOUT} seconds by default.'
                 ),
-            click.option(
+            option(
                 '--provision-tick', metavar='SECONDS',
                 help=f'How often check Artemis API for provisioning status, '
                      f'{DEFAULT_PROVISION_TICK} seconds by default.',
                 ),
-            click.option(
+            option(
                 '--api-timeout', metavar='SECONDS',
                 help=f'How long to wait for API operations to complete, '
                      f'{DEFAULT_API_TIMEOUT} seconds by default.',
                 ),
-            click.option(
+            option(
                 '--api-retries', metavar='COUNT',
                 help=f'How many attempts to use when talking to API, '
                      f'{DEFAULT_API_RETRIES} by default.',
                 ),
-            click.option(
+            option(
                 '--api-retry-backoff-factor', metavar='COUNT',
                 help=f'A factor for exponential API retry backoff, '
                      f'{DEFAULT_RETRY_BACKOFF_FACTOR} by default.',
```

Comment:
What is the reason why 2 more options below this line persist with `click.option()` instead of `tmt.options.option()`? I just `grep -R . 'click\.option'` through this branch.

Reply:
+1, looks like an incomplete sed

Reply:
```
examples/plugins/provision.py
32:            click.option(
35:            click.option(

tmt/options.py
251:    click.option(
255:    click.option(
262:    click.option(
269:    click.option(
276:    click.option(
280:    click.option(

tmt/steps/provision/artemis.py
555:            click.option(
560:            click.option(
```

seems these all occurances were not changed

Reply:
Correct, fixed in 017a1f73.